### PR TITLE
Hw9 scripting&serialization

### PR DESCRIPTION
**Pull request by Ionel14**

### CODE REVIEW

File: client/message.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+// namespace boost{
+//     namespace serialization{
+//         template<class Archieve>
+//         void serialize(Archieve &a, smart_home_client::Message &m, const unsigned version)
+//         {
+//             a & m.sender_ & m.date_ & m.text_;
+//         }
+//     }
+// }
```
Comment (CristianIonutIonescu) [inline client/message.h:41]: is this still needed? or can it be removed?
Please don't leave commented code, might confuse the readers along the way. If you don't need the code remove it.

**Review by CristianIonutIonescu — APPROVED** [review]
Homework looks ok.
Just a piece of advice:
I suggest to keep the smarthome files here for future development.

**Comment by Ionel14** [comment]
> Homework looks ok. Just a piece of advice: I suggest to keep the smarthome files here for future development.

I thought that I shouldn't keep the old smarthome files because I have them in the server project, so, in my opinion, they are useless there. But If I really need them, I can get them from the old pull requests.

File: client/message.h (hunk 1 above)
Comment (Ionel14) [reply client/message.h:41]: I forgot to delete it, I will pay more attention in the future

**Pull request merged**